jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/WarProjectIntegrationTest.java

```diff
@@ -49,6 +49,7 @@ public void testBuild_jettyServlet25() throws IOException, InterruptedException,
   @Test
   public void testBuild_tomcatServlet25()
       throws IOException, InterruptedException, DigestException {
+    // jkljlk
```

Comment:
Is this an intentional change?

Reply:
lol no, thank you for saving me from myself